tests/api2/test_alert.py

```diff
@@ -0,0 +1,75 @@
+from time import sleep
+
+import pytest
+
+from auto_config import pool_name
+from middlewared.test.integration.utils import call, ssh
+
+
+ID_PATH = "/dev/disk/by-partuuid/"
+
+
+def get_alert_by_id(alert_id):
+    return next(filter(lambda alert: alert["id"] == alert_id, call("alert.list")), None)
+
+
+def wait_for_alert(timeout=120):
+    for _ in range(timeout):
+        for alert in call("alert.list"):
+            if (
+                alert["source"] == "VolumeStatus" and
+                alert["args"]["volume"] == pool_name and
+                alert["args"]["state"] == "DEGRADED"
+            ):
+                return alert["id"]
+        sleep(1)
+
+
```

Comment:
Why `autouse` if then used explicitly?

Reply:
Just signifies clearly that it is a dependency for the whole module, even for hypothetical tests added in the future.

Reply:
Fair point